### PR TITLE
Add documentation for PbeParameters

### PR DESCRIPTION
**Pull request by bartonjs**

### CODE REVIEW

**Comment by bartonjs** [comment]
cc @rpetrusha @mairaw @carlossanlop

File: xml/System.Security.Cryptography/PbeParameters.xml [hunk 1]
```diff
@@ -17,8 +17,9 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents parameters to be used for Password-Based Encryption (PBE).</summary>
     <remarks>To be added.</remarks>
+    <threadsafe>All public and protected members of <see cref="T:System.Security.Cryptography.PbeParameters" /> are thread-safe and may be used concurrently from multiple threads.</threadsafe>
```
Comment (carlossanlop) [inline xml/System.Security.Cryptography/PbeParameters.xml]: First time I see a `<threadsafe>` item in docs. Some questions:

1. Does it show up in IntelliSense?
2. If it does not, is it important enough to be moved to the summary, or should it be moved under remarks?

**Comment by carlossanlop** [comment]
Aside from my questions regarding `<threadsafe>`, it all LGTM.

File: xml/System.Security.Cryptography/PbeParameters.xml (hunk 1 above)
Comment (bartonjs) [reply xml/System.Security.Cryptography/PbeParameters.xml]: As far as I know (I copied it from ConcurrentBag)

1) Yes
2) No
3) N/A (It's a distinct section on the docs page) -- https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentbag-1?view=netframework-4.8#thread-safety

Comment (mairaw) [reply xml/System.Security.Cryptography/PbeParameters.xml]: Yep, @bartonjs's correct

File: xml/System.Security.Cryptography/PbeEncryptionAlgorithm.xml [hunk 2]
```diff
@@ -40,7 +40,7 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates that encryption be performed with the AES-128 algorithm in CBC mode with PKCS#7 padding.</summary>
```
Comment (mairaw) [inline xml/System.Security.Cryptography/PbeEncryptionAlgorithm.xml]: Would it read better like the following?

```suggestion
        <summary>Indicates the encryption should be performed with the AES-128 algorithm in CBC mode with PKCS#7 padding.</summary>
```

File: xml/System.Security.Cryptography/PbeParameters.xml (hunk 1 above)
Comment (mairaw) [reply xml/System.Security.Cryptography/PbeParameters.xml]: Also, I think we have a request to bring the default one back when one is not provided like we used to have on MSDN

File: xml/System.Security.Cryptography/PbeEncryptionAlgorithm.xml (hunk 2 above)
Comment (bartonjs) [reply xml/System.Security.Cryptography/PbeEncryptionAlgorithm.xml]: I could go either way.  Right now we don't have anything that would return one of these values (other than asking PbeParameters what you had already saved), but we theoretically could in the future, so I was having voice issues.

Yours definitely reads more like a sentence someone would say, so let's go with your way :smile:.

**Pull request merged**